mne/time_frequency/spectrum.py

```diff
@@ -1420,6 +1510,11 @@ def average(self, method="mean"):
                 "supported. Consider averaging the signals before computing "
                 "the Welch spectrum estimates."
             )
+        if "taper" in self._dims:
+            raise NotImplementedError(
+                "Averaging multitaper tapers across epochs is not supported. Consider "
+                "averaging the signals before computing the complex spectrum."
+            )
```

Comment:
In the `EpochsSpectrum.average()` method, this is how complex multitaper data (with unaggregated tapers) was handled before support was removed, in the same way that Welch data with multiple segments is handled.

Reply:
OK with me. Until someone works through the maths to figure out the right way to do it, we should probably do nothing.